### ISSUE TITLE
Remove TestResult from xxxPropertyType

### ISSUE THREAD

**Issue by CharlieG021163** [ETPartners/b2mmlv7beta#7]
Related to ##16, #101
### MESA 
https://github.com/MESAInternational/B2MML-BatchML/issues/44

**Background**
1. B2MML 7.0 still has the TestResult element in each instance resource property type per the old version of ISA-95 resource models and B2MML 6.0.  The old resource model had an association relationship from each resource property to the Test Result object which did not allow Test Result object to be exchanged and queried as a 1st class object and did not allow Test Result object to be related to resource actuals.  

2. The updated 950002 as part of the collaboration with OAGi changed Test Results to a 1st order object as part of the Operations Test Model. 

3. ADDED new OperationsTest.xsd schema #101 with TestSpecificationType and TestResultType which allows 1st order transactions for both types.  I do not believe that the 1st order transactions for both types were added to the updated 950005.  

4. B2MML 7.0 beta only has B2MML-TestSpecification.xsd schema and has complex types of "TestResultType" and TestPropertyMeasurementType in the Common Schema.  

5. In the new Operations Test Model in updated 950002, Test Result object is a 1st order object that is able to be exchanged/queried and has relationship to resource actual which is represented as a root element.  

6.  The MESA Committee may reject changing their implementation.  If so, ERDi should create per updated standard.  

**Supporting Documents**
1. Clause 5.9.1 Operations test model

2. Each of following resource models in updated 95002 do not have a Test Result object in their resource property objects.  In the Operations Test Model, each of following resource models do have a Testable Object object that are target role in an association relationship from the Test Result object.
Clause 5.4.1 Personnel model
Clause 5.5.1 Role-based equipment model
Clause 5.6.1 Physical asset model
Clause 5.7.1 Material model

3. The following resource actual objects have a Test Result attributes per the Operations Test model
Clause 6.3.6 Personnel actual, Table 208 – Personnel actual relationship roles
Clause 6.3.8 Equipment actual, Table 212 – Equipment actual relationship roles
Clause 6.3.10 Physical asset actual, Table 216 – Physical asset actual relationship roles
Clause 6.3.12 Material actual, Table 220 – Material actual relationship roles

**Impacted Types and Solutions**
B2MML - Equipment.xsd, EquipmentPropertyType
B2MML - Material.xsd, MaterialLotPropertyType
B2MML - PhysicalAsset.xsd, PhysicalAssetPropertyType
B2MML - Personnel.xsd, PersonPropertyType

DELETE: TestResult element

**Comment by Dennis-Brandl** [comment]
I agree, there is no relationship in the ISA 05 Part 2 Ed2 document to the properties.  
Delete TestResult from complexType name =
"MaterialLotPropertyType", "EquipmentPropertyType", "PhysicalAssetPropertyType", "PersonPropertyType",

**Comment by terryrankine** [comment]
unsure if there is more work....

**Comment by gerardmoloneyetpartners** [comment]
@Dennis-Brandl so this change is accepted by MESA then? Please make it obvious by stating accepted.

**Comment by Dennis-Brandl** [comment]
Change not accepted.  Deleted TestResult from complexType name =
"MaterialLotPropertyType", "EquipmentPropertyType", "PhysicalAssetPropertyType", "PersonPropertyType".
